scalameta/tests/jvm/src/test/resources/semanticdb.expect

```diff
@@ -0,0 +1,26 @@
+scalahost/integration/src/main/scala/example/Example.scala
+----------------------------------------------------------
+Dialect:
+Scala212
```

Comment:
2.12 was released quite a while ago, so it's only fair that we use it as default. Also see https://github.com/scalameta/scalameta/issues/1011.